contrib/README.md

```diff
@@ -0,0 +1,137 @@
+# Community Driven Docker Examples
+These docker containers are meant to serve as an isolated develop/deployment environment. Each docker container has the 
+terraform libvirt provider built and placed in the custom plugins folder. The most common use case for these containers 
+is to run your terraform environment in a isolate container talking to a `remote` libvirt system. 
+
+Please refer to the distro's `README.md` for specific instructions.
+
+Additionally, you will find a `Build` folder along side the distro folders. The docker containers within the `Build`
+folder will compile the terraform libvirt provider for you. Please refer to the `Build` folder's `README.md` for more 
+information and instructions.
+
+## Table of Content
+- [Alpine Containers](Alpine/)
+- [Build Containers](Build/)
```

Comment:
I think the `build` containers should be seperate from the distro containers.

I would create 
## Table of content
### distro containers
- bla bla
### build containers